drivers/spi/spi_ll_stm32.h

```diff
@@ -53,6 +53,18 @@ struct spi_stm32_data {
 #endif
 };
 
+#ifdef CONFIG_SPI_STM32_DMA
+static inline uint32_t ll_func_spi_dma_tx(SPI_TypeDef *spi)
```

Comment:
Function name is not specific enough. Can you come up with a more meaningful name? (busy, empty, ...)

Reply:
renaming to "ll_func_spi_dma_busy"